meson.build

```diff
@@ -2895,14 +2899,37 @@ meson_install_args = meson_args + ['install'] + {
     'muon': []
 }[meson_impl]
 
+# since install_test_files are depending tmp_install
+# be sure about installing test files later
+tmp_install_priority = 100
 test('tmp_install',
     meson_bin, args: meson_install_args ,
     env: {'DESTDIR':test_install_destdir},
-    priority: 100,
+    priority: tmp_install_priority,
     timeout: 300,
     is_parallel: false,
     suite: ['setup'])
 
+# get full paths of test_install_libs to copy them
+test_install_libs_fp = []
+foreach lib: test_install_libs
+  test_install_libs_fp += lib.full_path()
+endforeach
+
+install_test_files_wrapper = files('src/test/install_test_files_wrapper')
+test('install_test_files',
+    python, args: [
+      install_test_files_wrapper,
+      '--install_path_files', '@0@/@1@/@2@'.format(
+        test_install_destdir, dir_prefix, contrib_data_args['install_dir']),
```

Comment:
Probably better to set this via the '/' operator, i.e. `test_install_destdir / dir_prefix / contrib_data_args['install_dir']`

Reply:
Seems like naming the `install_path_*` arguments `libdir` and `sharedir` would be easier to understand?

Reply:
> Probably better to set this via the '/' operator, i.e. test_install_destdir / dir_prefix / contrib_data_args['install_dir']

When I do that, it changes paths to relative paths. Is there any way to prevent that?

Reply:
Oh, it should be `test_install_location / ...` - that should fix that issue...